flask_resty/view.py

```diff
@@ -846,6 +846,29 @@ def update_item_raw(self, item, data):
             setattr(item, key, value)
         return item
 
+    def upsert_item(self, id, data, with_for_update=False):
+        """Updates an existing item with the matching id or if the item does not exist yet, create and insert it.
```

Comment:
nitpick: per PEP8, use imperative in docstrings

```suggestion
        """Update an existing item with the matching id or if the item does not exist yet, create and insert it.
```

Reply:
just gonna make this change cuz it's literally one character